### ISSUE TITLE
Focus on nav

### ISSUE THREAD

**Issue by katknow** [omeka-s-themes/cozy#39]
Currently, this theme is failing the Focus test for Accessibility Insights for Web. Currently, after the nav is opened and tabbed through it does not close after the final tab. Per the WCAG 2.4.3 Focus Order requirement, after tabbing through the menu it should return you to the original trigger component.

**Comment by kimisgold** [comment]
Cozy needed an update for its offcanvas menu, so it's been refactored. Now when tabbing, it tabs to a "close" button after the last item, which is appropriate for an offcanvas modal.

**Comment by aviannamiller** [comment]
There are a few inconsistencies happening with VoiceOver across browsers, but generally speaking, the "close" button is functional. I'm going to break the issues down by browser.

Chrome:
- Upon opening the menu, focus moves to the empty banner (which is also read out), while is the menu is now open and overlaying that content. Beyond that, tabbing and closing all function as expected, and focus is returned to the menu button upon closing the menu.

Safari:
- Opening and tabbing through the menu function as expected. However, closing the menu prompts a page reload, which of course redirects focus to the "Skip to main content" button instead of the initial trigger.

Firefox:
- Once the menu button is opened, VoiceOver announces the webpage, which makes me think focus is immediately moving to the web content as a whole instead of the menu. I am including a screenshot showing what happens as soon the menu button is activated: <img width="1280" alt="VoiceOver read out that says 'Welcome • Testing • Testing, web content' overlaying an Omeka S page with an open menu bar" src="https://github.com/user-attachments/assets/39458168-b5d1-40f1-a607-695512d2dde8" /> From there, however, there are no issues tabbing through, closing the menu, and returning focus to the menu button.

This definitely warrants further testing via NVDA and JAWS.

**Comment by aviannamiller** [comment]
Revisiting this issue after testing with NVDA and JAWS:

While close functionality technically works via screen reader, there really should be a visible "close" menu button of some sort. Currently, the `oc-container` region has the close functionality assigned to it, which works across screen readers, but this is not ideal. Screen readers can access this information, but it is just generally confusing for keyboard and mouse users on how to close out of the menu, especially since the visible menu toggle that is already part of the UI shifts off screen once the menu is opened.

A bit out of scope but relevant all the same: the menu `nav` element contains an ARIA role (role="dialog") that is not allowed on this type of element.
